app/services/ConsignmentService.scala

```diff
@@ -100,4 +102,11 @@ class ConsignmentService @Inject()(val graphqlConfiguration: GraphQLConfiguratio
     sendApiRequest(updateConsignmentSeriesIdClient, updateConsignmentSeriesId.document, token, variables)
       .map(data => data.updateConsignmentSeriesId.isDefined)
   }
+
+  def getConsignmentExport(consignmentId: UUID, token: BearerAccessToken): Future[getConsignmentForExport.GetConsignment] = {
```

Comment:
Should add some unit tests for this new method to the `ConsignmentServiceSpec.scala` as per the other methods.

Reply:
I've added some test for the getConsginmentExport method